code/game/machinery/transportpod.dm

```diff
@@ -0,0 +1,121 @@
+/obj/machinery/transportpod
+	name = "Ballistic Transportation Pod"
+	desc = "A fast transit ballistic pod used to get from one place to the next. Batteries not included!"
+	icon = 'icons/obj/structures.dmi'
+	icon_state = "borg_pod_opened"
+
+	density = 1 //thicc
+	anchored = 1
+	use_power = 0
+
+	var/used = 0
```

Comment:
Rather than have to check this var, wouldn't it just be simpler to move the mob and destroy this...?

Reply:
That would be replacing a simple 'used' variable with some code to delete the old one. Just kind of how I thought it would be easier. The used variable also helps with changing the description of the pod.

Reply:
If it's destroyed when it's done doing its thing you don't need to worry about that.

Reply:
I mean.. You want it to be destroyed after it drops?

Reply:
After it drops and spits out the occupant, yes.